sdk-api-src/content/debugapi/nf-debugapi-isdebuggerpresent.md

```diff
@@ -73,9 +73,6 @@ This function allows an application to determine whether or not it is being debu
 
 To determine whether a remote process is being debugged, use the <a href="/windows/desktop/api/debugapi/nf-debugapi-checkremotedebuggerpresent">CheckRemoteDebuggerPresent</a> function.
 
-To compile an application that uses this function, define the _WIN32_WINNT macro as 0x0400 or later. For more information, see 
-<a href="/windows/desktop/WinProg/using-the-windows-headers">Using the Windows Headers</a>.
-
```

Comment:
> [...] the bottom of the page mention the minimum OS requirement anyway.

The documentation indicates `0x0400` (Windows NT 4), but the bottom of the page says Windows XP / Server 2003.

(The metadata at the bottom of pages is typically incorrect, making this sentence much more valuable.)

Reply:
Interesting, indeed confirmed the current metadata is false, screenshot from msdn.microsoft.com from 2004 wayback machine:
![image](https://github.com/user-attachments/assets/d050faa2-541d-4919-b17f-16a035da7ec5)

will work on a fix for this

Reply:
fixed